### PR TITLE
Update init_fernet.yml

### PR DESCRIPTION
**Pull request by codyhammock**
Generate fernet token files from keystone_fernet_token_list, supplied in passwords.yml

### CODE REVIEW

File: ansible/roles/keystone/tasks/init_fernet.yml [hunk 1]
```diff
@@ -17,31 +39,10 @@
   retries: 10
   delay: 5
   run_once: True
-  delegate_to: "{{ groups[keystone_services['keystone-fernet'].group][0] }}"
-  when: not use_preconfigured_fernet | bool
+  delegate_to: "{{ groups['keystone'][0] }}"
```
Comment (diurnalist) [inline ansible/roles/keystone/tasks/init_fernet.yml]: Better to keep this the way it was before; assuming there is a `keystone` group seems sane but it makes a bit less flexible for people who are overriding things here (assuming we want to propose this upstream.)

File: ansible/roles/keystone/tasks/init_fernet.yml [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+    keystone_fernet_tokens_mountpoint: "{{ dataresult[0].Mountpoint }}"
+
+- name: Create fernet token file(s)
+  become: True
+  copy:
+    dest: "{{ keystone_fernet_tokens_mountpoint }}/{{index}}"
+    content: "{{item}}"
+  loop: "{{ keystone_fernet_token_list }}"
+  loop_control:
+    index_var: index
```
Comment (diurnalist) [inline ansible/roles/keystone/tasks/init_fernet.yml]: What is the default value of `index_var`? Is it not `index`?

Comment (codyhammock) [reply ansible/roles/keystone/tasks/init_fernet.yml]: Evidently not. I had to add that declaration explicitly. 🤷‍♂️

File: ansible/roles/keystone/tasks/init_fernet.yml (hunk 1 above)
Comment (codyhammock) [reply ansible/roles/keystone/tasks/init_fernet.yml]: This was the delegation in the original, but your suggestion seems sensible.

**Pull request merged**